10-task/dubbo-samples-extensibility/dubbo-samples-extensibility-filter-provider/src/main/resources/application.yml

```diff
@@ -0,0 +1,31 @@
+#   Licensed to the Apache Software Foundation (ASF) under one or more
+#   contributor license agreements.  See the NOTICE file distributed with
+#   this work for additional information regarding copyright ownership.
+#   The ASF licenses this file to You under the Apache License, Version 2.0
+#   (the "License"); you may not use this file except in compliance with
+#   the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+dubbo:
+    application:
+        # Specify the application name of Dubbo
+        name: extensibility-filter-provider
+    protocol:
+        # Specify the port of Dubbo protocol
+        port: 20881
+    provider:
+        # Apply AppendedFilter
+        filter: appended
+        # Enable token verification for each invocation
+        token: true
+    registry:
+        # Specify the registry address
+        address: nacos://localhost:8848?username=nacos&password=nacos
+#        address: nacos://localhost:8848?username=nacos&password=nacos
```

Comment:
Is this configuration file only a new, no-removed properties file?

Reply:
No, this is also replacing `application.properties` with `application.yml`
old:
![image](https://github.com/apache/dubbo-samples/assets/91261012/3190b34e-468a-4226-90d2-4d7e19929526)
![image](https://github.com/apache/dubbo-samples/assets/91261012/534e1a55-5825-4b6f-a197-9a13c0ab2eae)
new:
![image](https://github.com/apache/dubbo-samples/assets/91261012/5a760cd5-adbe-463c-8709-19ff66041b67)

Reply:
@cnzakii My meaning is we should remove the application.properties.

Reply:
@CrazyHZM 
My reply above is explaining that I have deleted `application.properties`.
But I don’t know why it is not shown here that `application.properties` has been deleted

Reply:
Ok, I checked your branch, and it is ok; it seems to be a GitHub issue.